modules/check_in/spec/services/v2/chip/service_spec.rb

```diff
@@ -160,4 +160,56 @@
       end
     end
   end
+
+  describe '#demographic_confirmations' do
+    Timecop.freeze(Time.zone.now)
```

Comment:
Thanks for covering specs for the changes.  Came across this [doc](https://www.rubydoc.info/gems/liangzan-timecop/Timecop.freeze) earlier on `Timecop.freeze` changing system time.  

Could you please use code block while using Timecop as in [this spec](https://github.com/department-of-veterans-affairs/vets-api/blob/643c77ebd2f069f3915896cc93beccda09da374c/modules/check_in/spec/services/v2/lorota/redis_client_spec.rb#L80-L82)?  This will prevent system time change when the spec runs.

Reply:
I will go ahead and make the change so that we're using the block instead since this makes it clearer for someone reading the test. However, if you look toward the bottom, you can see that I'm returning `Timecop.return` which will put the time back.

Reply:
Thanks for updating Timecop to use blocks @dillo